source/NVDAObjects/__init__.py

```diff
@@ -1191,5 +1191,5 @@ def getSelectedItemsCount(self,maxCount=2):
 		Fetches the number of descendants currently selected.
 		For performance, this method will only count up to the given maxCount number, and if there is one more above that, then sys.maxint is returned stating that many items are selected.
 		"""
-		return 0
+		return 1

```

Comment:
I think returning 1 should some how be moved in to specific support for LibreOffice..